LEGO1/lego/legoomni/src/act1/act1state.cpp

```diff
@@ -1,34 +1,287 @@
 #include "act1state.h"
 
+DECOMP_SIZE_ASSERT(Act1State, 0x26c)
+DECOMP_SIZE_ASSERT(Act1State::NamedPlane, 0x4c)
+DECOMP_SIZE_ASSERT(Act1State::NamedTexture, 0x14)
+
+// Forward declarations
+Act1State::NamedTexture* ReadNamedTexture(LegoFile* p_file);
```

Comment:
Those three functions are very likely part of the `LegoUtil` collection. You can tell by their address.